### PR TITLE
Fix ordering expression for axes

### PR DESCRIPTION
**Pull request by zBritva**
Fix for https://github.com/microsoft/charticulator/issues/1052, https://github.com/microsoft/charticulator/issues/704#issuecomment-1320496691, https://github.com/microsoft/charticulator/issues/1012

### CODE REVIEW

**Comment by jdbiac** [comment]
I hope this can be released soon! Thanks once again zBritva for fixing this!

File: src/app/stores/app_store.ts [hunk 1]
```diff
@@ -1962,7 +1966,17 @@ export class AppStore extends BaseStore {
               dataExpression.valueType,
               values
             );
-            dataBinding.orderByCategories = deepClone(categories);
+            try {
+              if (dataBinding.orderByExpression) {
+                dataBinding.orderByCategories = this.getCategoriesForOrderByColumn(
+                  dataBinding
+                );
+              } else {
```
Comment (MatthLee) [inline src/app/stores/app_store.ts:1974]: consider merging this same logic (the else and catch) into a single finally block

**Review by MatthLee — APPROVED** [review]

**Comment by jdbiac** [comment]
Hi, what step is pending for this change to be released? Is there any due date?

**Comment by jdbiac** [comment]
Hi @zBritva no new updates on this? What is missing?

**Comment by zBritva** [comment]
@jdbiac I don't have permissions to merge PRs. And I also can't publish new releases

**Comment by jdbiac** [comment]
Is it @MatthLee who can approve it and merge it to publish a new release?

**Comment by jdbiac** [comment]
Hi, can someone close this? Is it too much asking this for Christmas :). I appreciate your help!

**Comment by zBritva** [comment]
@jdbiac, [in progress](https://github.com/zBritva/charticulator-visual-community)

**Pull request closed without merging**